index.html

```diff
@@ -1,164 +1,197 @@
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous" />
+  <!-- custom style -->
+  <link rel="stylesheet" href="css/pattern.css" />
+  <link rel="stylesheet" href="css/timeline.css" />
+  <link rel="stylesheet" href="css/main.css" />
+  <!-- for Facebook -->
+  <meta property="og:title" content="JIIT ICABB 2021" />
+  <meta property="og:type" content="website" />
+  <meta property="og:image:secure" content="https://icabbjiit.com/assets/meta_img.png" />
+  <meta property="og:url" content="https://icabbjiit.com/" />
+  <meta property="og:description" content="JIIT ICABB 2021 is the biggest 2021 digital conference held virtually from January 28th to January 30th 2021" />
+</head>
+
+<body>
+  <lines>
+    <div class="wrap"></div>
+    <div class="wrap"></div>
+    <div class="wrap"></div>
+    <div class="wrap"></div>
+    <div class="wrap"></div>
+  </lines>
+  <content>
+    <div class="header">
+      <img class="bg-main" src="assets/Live-Wave-Background.svg" alt="" />
+      <div class="shape shape-1">
+        <img src="img/blue/blueShape1.png" />
       </div>
-      <div class="main">
-        <div class="main-cent animated fadeInDown">
-          <div class="container">
-            <p class="header-text" style="font-size: 1.5rem" href="#sponsors">
-              JAYPEE INSTITUTE OF INFORMATION TECHNOLOGY
-            </p>
-            <p class="header-text" style="font-size: 1.5" href="#sponsors">
-              IS
-            </p>
-            <p class="header-text" style="font-size: 1.5" href=" #sponsors">
-              ORGANISING
-            </p>
-          </div>
-          <img class="logo" src="assets/logo.png" alt="" />
-          <div class="main-header-div">
-            <p style="margin-bottom: 0; font-size: 1.4rem">
-              INTERNATIONAL CONFERENCE
-            </p>
-            <p style="margin-bottom: 0; font-size: 1.4rem">ON</p>
-            <p style="margin-bottom: 0; font-size: 1.4rem">
-              ADVANCES IN BIOSCIENCES AND BIOTECHNOLOGY - (ICABB)
-            </p>
-            <p style="margin-bottom: 0; font-size: 1.4rem">2021</p>
-          </div>
-          <a href="registration"><button>REGISTER</button></a>
-          <a href="guidelines"><button>GUIDELINES</button></a>
-          <a href="ICABB-2021-Brochure.pdf" target="_blank"
-            ><button>VIEW BROCHURE</button></a
-          >
-          <a href="schedule.html"><button>SCHEDULE</button></a>
-          <a href="JournalPartner.html"><button>JOURNAL PARTNERS</button></a>
-          <p class="power" href="#sponsors">
-            28<sup>th</sup> January - 30<sup>th</sup> January 2021
+      <div class="shape shape-2">
```

Comment:
Why removed main and container classes?

Reply:
I didn't touch these classes except for in navbar, because they didn't want separate buttons, but now I have added the container class to the navigation div and its working fine, however is there a way that the width of the bar always remains 100% no matter the width of the screen?